pennylane/resource/specs.py

```diff
@@ -58,13 +58,15 @@ def specs(qnode, max_expansion=None, expansion_strategy=None):
         def circuit(x, add_ry=True):
             qml.RX(x[0], wires=0)
             qml.CNOT(wires=(0,1))
+            qml.TrotterProduct(hamiltonian, time=1.0, order=2)
```

Comment:
Might be good to have another operation with error, to make sure they are combined properly.